grails-app/services/org/bbop/apollo/TrackService.groovy

```diff
@@ -1293,4 +1305,310 @@ class TrackService {
         return organismJBrowseDirectory
     }
 
+    /**
+     *
+     * @param trackObject
+     * @param file
+     * @return
+     */
+    def parseTracksConf(JSONObject trackObject, File file) {
+        ArrayList sectionArray = new ArrayList()
+        ArrayList keypathArray = new ArrayList()
+        String operation, value
+        HashMap confMap = [:]
+
+        file.text.split("\n").each { line ->
```

Comment:
I think you can use eachLine instead of split here: http://mrhaki.blogspot.com/2009/11/groovy-goodness-working-with-lines-in.html  

Not a big deal.